reference/cd-gitlab-ci.md

```diff
@@ -0,0 +1,183 @@
+---
+title: GitLab CI
+---
+
+This page details how to use [GitLab CI](https://about.gitlab.com/features/gitlab-ci-cd/) to manage deploying
+staging and production stacks based on commits to specific Git branches. This is sometimes
+referred to as Push-to-Deploy.
+
+Pulumi doesn't require any particular arrangement of stacks or workflow to work in a
+continuous integration / continuous deployment system. So the steps described here can be
+altered to fit into any existing type of deployment setup.
+
+## Prerequisites
+- An account on https://app.pulumi.com and that you have created a new project.
+  - As of this writing, we support logins only using GitHub identity.
+  - This just means you will sign-in using your GitHub credentials.
+  - However, pulumi can be run from anywhere and your infrastrucutre code itself can be hosted anywhere.
+- The latest CLI. Installation instructions are [here](https://pulumi.io/quickstart/install.html).
+- A bare repo and set the remote URL to be your GitLab repo.
```

Comment:
Would it make more sense to say "a GitLab project, with GitLab CI/CD enabled" or something like that? The repo and remote will be set up automatically when GitLab's CI system runs, correct?

Reply:
CI/CD feature is automatically enabled as long as the project has a `.gitlab-ci.yml` file in the root (default location) or they can specify the search path for that file if it is in a different location, in their GitLab project's settings. I'll revise this to use the GitLab's terminology though.